src/content/docs/infrastructure/host-integrations/host-integrations-list/install-apache-flink.mdx

```diff
@@ -1,10 +1,12 @@
 ---
```

Comment:
Hi @akristen! I think that it should be better "Apache Flink dashboard" than "Apache Flink dashboards", in singular if we follow the [quickstart page](https://newrelic.com/instant-observability/apache-flink). I also think that it should be always "Apache Flink" and not Flink only.